src/app.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const terminal = require('./modules/terminal');
+const fs = require('fs');
+
+function copyFile() {
+  terminal.question('Enter your command: ', (key) => {
```

Comment:
It's good practice to trim the user input to remove unnecessary spaces at the beginning and end.